packages/utils/src/fetchWithTimeout.ts

```diff
@@ -0,0 +1,16 @@
+/** Set custom timeout for fetch. Default value is set to 8000. */
+export const fetchWithTimeout = async (
+    resource: RequestInfo | URL,
+    options: RequestInit & { timeout?: number } = {},
+) => {
+    const { timeout = 8000 } = options;
+
+    const controller = new AbortController();
+    const id = setTimeout(() => controller.abort(), timeout);
+    const response = await fetch(resource, {
```

Comment:
The package `utils` is suppose to contain utilities that work for both browser and nodejs. As far as I know `fetch` is added to nodejs 17 as experimental feature, but we are still in nodejs 16 in this project.

Maybe we could call it `browserFetchWithTimout` or something like that to avoid confusion, and leave a `todo` to make it generic once we have fetch in nodejs fully.

Reply:
good point, I guess it makes sense to use `cross-fetch` here

Reply:
I think it is important to keep `utils` package as dependency free as possible, there is 0 dependencies right now, only dev dependencies.
I like the idea of just changing the name and using it only in browser because I know what happens with exceptions, with time they become the normal.

But I don't want to be very picky here and I guess we could make an exception for `cross-fetch` if you consider it to be important.

Reply:
@marekrjpolak advised me, that we already have even more powerful util `scheduleAction` that could be used here and we don't need this new util at all 🎉 
beb462cd56f8bae493c584a672a55ef5c475d482